src/console_reporter.cc

```diff
@@ -92,7 +95,9 @@ void ConsoleReporter::PrintRunData(const Run& result) {
                    " items/s");
   }
 
-  double const multiplier = 1e9; // nano second multiplier
+  double const multiplier = time_unit_ == "ns" ? 1e9 : 1e3; // nano second or
```

Comment:
i don't think anyone is parsing the console output so just changing it should be fine.

we have csv and json that is more readily parseable, and shouldn't be affected by this change.